bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookieWatcherImpl.java

```diff
@@ -359,5 +359,9 @@ public void quarantineBookie(BookieId bookie) {
         }
     }
 
-
+    public void releaseAllQuarantinedBookies(){
+        if (quarantinedBookies.size() > 0){
```

Comment:
why do you need this condition?

Reply:
This judgment seems superfluous and has been removed.